app/controllers/mixins/containers_common_logging_mixin.rb

```diff
@@ -0,0 +1,39 @@
+module ContainersCommonLoggingMixin
+  def launch_common_logging
+    record = self.class.model.find_by_id(params[:id])
+    ems = record.ext_management_system
+    route_name = ems.common_logging_route_name
+    logging_route = ContainerRoute.find_by(:name => route_name, :ems_id => ems.id)
+    if logging_route
+      user_token = SecureRandom.base64(15)
```

Comment:
I was wondering if I should move this whole piece of code to the model, maybe to contianer_manager_mixin, any thoughts on that anyone?
Ideally this should be there but I also think it should be in a specific file for "common_logging" operations.